packages/calling/src/CallHistory/callHistoryFixtures.ts

```diff
@@ -439,3 +539,29 @@ export const ERROR_DETAILS_400 = {
   },
   message: 'FAILURE',
 };
+export const MOCK_LINES_API_CALL_RESPONSE: UCMLinesResponse = {
```

Comment:
Please leave a line before this. Also a comment for the above fixtures.

Reply:
Done